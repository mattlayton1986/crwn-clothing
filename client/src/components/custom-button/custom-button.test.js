import React from "react";
import { shallow } from "enzyme";
import CustomButton from "./custom-button.component";

describe("CustomButton component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CustomButton />);
  });

  it("should render CustomButton component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
