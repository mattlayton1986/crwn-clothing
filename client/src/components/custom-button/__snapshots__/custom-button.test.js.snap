// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton component should render CustomButton component 1`] = `ShallowWrapper {}`;
